Common/include/option_structure.hpp

```diff
@@ -2107,57 +2107,62 @@ static const MapType<string, ENUM_INPUT_REF> Input_Ref_Map = {
  * \brief Vertex-based quantities exchanged during periodic marker communications.
  */
 enum PERIODIC_QUANTITIES {
```

Comment:
... bonus points if you do it anyway

Reply:
rain check.
These enums can be almost entirely removed by implementing the comms such that they operate on entire containers.
Then the solver just passes the container instead of the enum. Periodics have extra logic but some cases can also be removed.
Too tired to do that now, though